package.json

```diff
@@ -45,9 +45,7 @@
     "url": "https://github.com/microsoft/react-native-test-app.git"
   },
   "scripts": {
-    "ci": "yarn --use-yarnrc .yarnrc-offline --prefer-offline --frozen-lockfile --non-interactive",
-    "clean": "npx --quiet rimraf example/node_modules/react-native-test-app && git clean -dfqx",
-    "deduplicate": "npx yarn-deduplicate --strategy fewer",
```

Comment:
we don't need dedup anymore because yarn 3 is better at it, right?

Reply:
It's somewhat better but the reason for removing this script is because it is now built in: https://yarnpkg.com/cli/dedupe